doc/src/manual/faq.md

```diff
@@ -194,6 +194,38 @@ c = 3::Int64
 If Julia were a language that made more liberal use of ASCII characters, the splatting operator
 might have been written as `...->` instead of `...`.
 
+### Is the last assigned variable always returned?
+
+No! The operator `=` always returns the right-hand side, therefore:
+
+```jldoctest
+function getthree()
+    x::Int = 3.
+    x # returns variable x
+end
+function getthreequick()
+    x::Int = 3. # returns 3.
+end
+
+typeof(getthree()) === Int
+typeof(getthreequick()) === Float64
+```
+
+and similarly:
+
+```jldoctest
+function threetup()
+    x,y = [3,3]
+    x,y # returns a tuple
+end
+threearr() = x,y = [3,3] # returns an array
+
+typeof(threetup()) === Tuple{Int,Int}
+typeof(threearr()) === Array{Int,1}
```

Comment:
use of `Int`  is preferred as it would work for both 23-bit and 64-bit versions of Julia.